src/layout/layout.rs

```diff
@@ -581,16 +592,15 @@ impl Layout {
         let spacing = self.spacing;
         let constraints = &self.constraints;
 
-        let area_size = Element::from((*variables.first().unwrap(), *variables.last().unwrap()));
-        configure_area(&mut solver, area_size, area_start, area_end)?;
-        configure_variable_constraints(&mut solver, &variables, area_size)?;
+        let area_size = area_end - area_start;
+        configure_variable_constraints(&mut solver, &variables, area_start, area_end)?;
```

Comment:
Can this still use the area approach and just make that [start*100,end*100]?
What makes this need to change to use floats here?

Reply:
It's adding an extra variable to the solver and the optimization program, which when debugging I wanted to make sure wasn't the cause of any instability, so I removed it. I do think it is cleaner that way but reverted the change now.